sbin/prepareWorkspace.sh

```diff
@@ -591,10 +591,14 @@ checkingAndDownloadingFreeType() {
     if ./configure --help | grep "with-png"; then
       pngArg="--with-png=no"
     fi
+    local harfbuzzArg=""
+    if ./configure --help | grep "with-harfbuzz"; then
+      harfbuzzArg="--with-harfbuzz=no"
```

Comment:
The only need to have this if is to distinguish jdk8 from others? (jsut more clever, to count possible future jdk8 with harfbuz..) . Right?

Reply:
No sure I understand the question. Whole function is [used for jdk 8 only](https://github.com/adoptium/temurin-build/blob/4ea25fa5e9ee5e7e77b66b6bf60bac3573c4c9f9/sbin/prepareWorkspace.sh#L853) (from alive ones). Newer jdk's (11+) have in-tree freetype.